girder/girder_large_image/rest/__init__.py

```diff
@@ -17,6 +19,7 @@ def addSystemEndpoints(apiRoot):
     """
     apiRoot.folder.route('GET', (':id', 'yaml_config', ':name'), getYAMLConfigFile)
     apiRoot.folder.route('PUT', (':id', 'yaml_config', ':name'), putYAMLConfigFile)
+    apiRoot.folder.route('PUT', (':id', 'items'), createLargeImages)
```

Comment:
This adds the endpoint `folder/:id/items`.  Other endpoints that are explicitly for large_images are either in `item/:id/tiles/...` or `large_image/...`.  I think, conceptually, this is a utility function so it would make more sense to move this to rest/large_image_resource.py endpoint and rename the route to `('folder', ':id', 'create')` (though I could also see using `tiles` rather than `create` for consistency).

Reply:
That route and location make more sense. Function moved and route renamed [here](https://github.com/girder/large_image/commit/588d03056eeedf6c4f06037d81e360a769e41140).